src/js/components/covid19/Heading.jsx

```diff
@@ -35,6 +34,18 @@ const Heading = () => {
                     </p>
                 </ReadMore>
             </div>
+            <div className="heading__img-wrapper">
+                <img
+                    className="heading__img"
+                    src="img/desktop@2x.png"
+                    alt="An illustration of a masked woman and child holding hands speaking to a masked doctor holding a tablet." />
+            </div>
+            <div style={{ marginLeft: "auto" }} className="heading__img-wrapper">
+                <img
+                    className="heading__img-mobile"
+                    src="img/mobile@2x.png"
+                    alt="An illustration of a masked woman and child holding hands speaking to a masked doctor holding a tablet." />
```

Comment:
508 stuff: purely decorative images shouldn't have alt text, just alt=""

Reply:
Got it thanks, removed alt text! 560px was also a specific edge case fix